### PR TITLE
Added onlyCountries attribute to filter available countries

### PR DESCRIPTION
**Pull request by crohit92**

### CODE REVIEW

**Comment by webcat12345** [comment]
Nice suggestion. Will check request and merge soon.

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "url": "https://github.com/webcat12345/ngx-intl-tel-input/issues"
   },
   "devDependencies": {
+    "@angular/cli": "^1.2.1",
```
Comment (webcat12345) [inline package.json]: angular-cli is not required at all.

File: package.json [hunk 2]
```diff
@@ -65,6 +66,7 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "@angular/forms": "^4.1.2"
+    "@angular/forms": "^4.1.2",
+    "@types/google-libphonenumber": "^7.4.11"
```
Comment (webcat12345) [inline package.json]: what is the usage of this library? and where you used this?

File: src/ngx-intl-tel-input.component.ts [hunk 3]
```diff
@@ -25,6 +28,12 @@ export class NgxIntlTelInputComponent implements OnInit {
   }
 
   ngOnInit() {
+
```
Comment (webcat12345) [inline src/ngx-intl-tel-input.component.ts]: what happens if user doesn't know ISO country string?

File: package.json (hunk 2 above)
Comment (crohit92) [inline package.json]: This library is to be used for mobile number validations, I didn't add that method in this pull request, but I will contribute that aswell.

File: src/ngx-intl-tel-input.component.ts (hunk 3 above)
Comment (crohit92) [inline src/ngx-intl-tel-input.component.ts]: When user knows which countries he want to display in the dropdown, (considering the user to be a aware developer) he can easily specify the iso codes of those countries and if he doesn't know he can google them and specify those.

File: package.json (hunk 1 above)
Comment (crohit92) [inline package.json]: I will remove that

**Pull request closed without merging**